effects/README.md

```diff
@@ -24,3 +22,305 @@ Amiga 500 knowledge base
 
 3. Subpixel line drawing using Blitter: [part 1](http://scalibq.wordpress.com/2011/12/28/just-keeping-it-real-part-5/) and [part 2](http://scalibq.wordpress.com/2012/01/06/just-keeping-it-real-part-5-1/).
 
+### MC68000 instruction timing
+
+Taken from [EAB post](https://eab.abime.net/showthread.php?t=103006).
+
+```
++---------------------+----+----+-----+-----+-----+------+--------+-----+-----+------+--------+----+----+----+----+
+|mnemonic |syntax     |dy  |ay  |(ay) |(ay)+|-(ay)|(d,ay)|(d,ay,r)|(m).w|(m).l|(d,pc)|(d,pc,r)|#d  |ccr |sr  |usp |
++---------+-----------+----+----+-----+-----+-----+------+--------+-----+-----+------+--------+----+----+----+----+
```

Comment:
The whole table could be rather easily converted to markdown for a nicer display

Reply:
Good idea, but let's move it to `demoscene.wiki` first.